deploy/olm-catalog/compliance-operator/manifests/compliance-operator.clusterserviceversion.yaml

```diff
@@ -1406,8 +1407,8 @@ spec:
   maintainers:
   - email: support@redhat.com
     name: Red Hat Support
-  maturity: alpha
+  maturity: stable
```

Comment:
what is the reasoning behind this change? Can we claim maturity stable even if we ship alpha APIs?

Reply:
uhm... good question! I honestly don't know. I consider the operator to be stable, and we've actually had quite stable APIs for a while. But you're right, maybe this is not appropriate cause we still have alpha CRDs.

Reply:
Fixed

Reply:
btw I couldn't find any explanation of what this flag means on operator-sdk homepage and oc explain just says it's a string..so I'm fine changing to stable since it just seems like an UI element. I filed https://issues.redhat.com/browse/CMP-986 so we can move our APIs beyond alpha.